connection.go

```diff
@@ -127,8 +143,17 @@ func (a *amqpConnection) open() error {
 
 	a.logger.Debug("Connecting to RabbitMQ server", logField{Key: "uri", Value: a.uriForLog()})
 
+	props := amqp.NewConnectionProperties()
+	if a.connectionName != "" {
+		props.SetClientConnectionName(a.connectionName)
+	}
+
 	// We request a connection from the RabbitMQ server.
-	conn, err := amqp.Dial(a.uri)
+	conn, err := amqp.DialConfig(a.uri, amqp.Config{
+		Heartbeat:  10 * time.Second,
+		Locale:     "en_US",
```

Comment:
We could probably move those values to `constants.go` file

```suggestion
		Heartbeat:  defaultHeartbeat,
		Locale:     defaultLocale,
```

Reply:
@m3talux Added a commit to do just that. Added them as a separate section since they're not really connection parameters.